src/Resources/config/mongodb.php

```diff
@@ -28,7 +28,12 @@
             ])
 
         ->set('sonata.user.listener.user', UserListener::class)
-            ->tag('doctrine_mongodb.odm.event_subscriber')
+            ->tag('doctrine.event_listener', [
```

Comment:
make sure the mongodb tag is like this please, with the subscriber it was different.